code/range/Model/src/main/java/ru/unn/agile/range/model/IntegerSet.java

```diff
@@ -0,0 +1,34 @@
+package ru.unn.agile.range.model;
+
+import java.util.StringJoiner;
+import java.util.stream.Stream;
+
+public class IntegerSet {
+    private int[] values;
```

Comment:
just a question: 
Why do you use array [] instead of List?
List is easier in your case, isn't it?

Reply:
method split generate String[] and it perfomance is better than List